Formula/orientdb.rb

```diff
@@ -64,7 +61,7 @@ def caveats
     sleep 4
 
     begin
-      assert_match "OrientDB Server v.2.2.5", shell_output("curl -I localhost:2480")
+      assert_match "OrientDB Server v.2.2.11", shell_output("curl -I localhost:2480")
```

Comment:
It does actually and links to http://www.rubydoc.info/github/Homebrew/brew/master/Formula#version-class_method. It's defined by default so you can do `assert_match "OrientDB Server v.#{version}"` without other changes.